backend/dal/Services/Concrete/ParcelService.cs

```diff
@@ -0,0 +1,100 @@
+using System;
+using System.Collections.Generic;
+using Pims.Dal.Entities;
+using System.Linq;
+using System.Security.Claims;
+
+namespace Pims.Dal.Services.Concrete
+{
+    //TODO: Comments
+    public class ParcelService : IParcelService
+    {
+        private readonly PIMSContext _dbContext;
+        private readonly ClaimsPrincipal _user;
+        public ParcelService(PIMSContext dbContext, ClaimsPrincipal user)
```

Comment:
We may not need to have a separate constructor variable for user, as we should have access to this within the context.